include/xrpl/server/detail/SSLHTTPPeer.h

```diff
@@ -164,7 +164,7 @@ SSLHTTPPeer<Handler>::do_handshake(yield_context do_yield)
         this->port().protocol.count("https") > 0;
     if (http)
     {
-        boost::asio::spawn(
+        ripple::util::spawn(
```

Comment:
```suggestion
        util::spawn(
```

`ripple::` can be removed